packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.html

```diff
@@ -0,0 +1,35 @@
+<template>
+    <div
+        id="listbox-id"
+        class="slds-dropdown slds-dropdown_length-5 slds-dropdown_fluid"
+        role="listbox">
+        <ul class="slds-listbox slds-listbox_vertical" role="presentation">
+            <template for:each={suggestionsToRender} for:item="suggestion">
+                <li key={suggestion.rawValue}
+                    role="presentation"
+                    class="slds-listbox__item"
+                    data-rawvalue={suggestion.rawValue}
```

Comment:
I think I was unaware we could access object attributes like this in the templating. Pretty cool

Reply:
really?

Reply:
Yeah well I don't remember ever using it so it doesn't look familiar.